databases/week2/hw-part2.sql

```diff
@@ -0,0 +1,28 @@
+-- task for creating the school database
+CREATE DATABASE school_db;
+
+USE school_db;
+
+CREATE TABLE class (
+    `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    `name` varchar(255) NOT NULL,
+    `begins` DATETIME NOT NULL,
+    `ends` DATETIME NOT NULL
+);
+
+CREATE TABLE student (
+    `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    `name` varchar(255) NOT NULL,
+    `email` varchar(255) NOT NULL,
+    `phone` varchar(255) NULL,
+    `class-id` int(10) unsigned NOT NULL,
```

Comment:
Good job with the new tables!

A tiny nitpick is that I'd suggest sticking to underscores (`_`) as `-` could be confused with subtraction and might even be an illegal identifier in some programming languages! 😄